src/task-registry/scheduledTasks.ts

```diff
@@ -73,6 +73,7 @@ export const SCHEDULED_TASKS: TaskSchedule[] = [
     task: syncNotifications,
     schedule: "27 * * * * *", // 27th second of every minute
     interval: moment.duration(1, "minutes").asMilliseconds(),
+    notification: "notification",
```

Comment:
did you mean `channel: "notification",` ?

Reply:
oh yeah haha